eb

```diff
@@ -82,5 +82,5 @@ then
     echo "Make sure you PYTHONPATH setting is correct."
     exit 1
 else
-    python $easybuild_main $@
+    python -O $easybuild_main $@
```

Comment:
supporting a way to turn of the use of `-O` might make sense...

maybe the wrapper should check whether `-d` or `--debug` was passed on the command line (may be tricky for `-d`...) + check whether `$EASYBUILD_DEBUG` is set? this doesn't catch the case where `debug` is enabled via a config file though...